code/game/objects/items/teleportation.dm

```diff
@@ -309,5 +309,180 @@
 			itemUser.visible_message(span_suicide("[user] looks even further depressed as they realize they do not have a head...and suddenly dies of shame!"))
 		return (BRUTELOSS)
 
+/obj/item/teleporter
```

Comment:
Rename this to syndicate_teleporter or something, there's a lot of things this typepath could be.

Reply:
Why was this marked as resolved? It's still here

Reply:
@JohnFulpWillard I mark stuff as resolved whenever I make the change on my laptop so I can keep track, I just haven't pushed it yet